pkgs/development/python-modules/curio/default.nix

```diff
@@ -22,6 +22,7 @@ buildPythonPackage rec {
   # test_aside_basic times out,
   # test_aside_cancel fails because modifies PYTHONPATH and cant find pytest
   checkPhase = ''
+    rm -r tests/__pycache__ # was packaged with this accidentally
```

Comment:
would you mind adding a link to the curio issue here? besides that LGTM

Reply:
oh right, will do

Reply:
pushed update

Reply:
@marsam take a look?